MIGRATION.md

```diff
@@ -1,3 +1,7 @@
 - Support for Scala 2.7, 2.8 and 2.9 has been dropped. [#86][86]
 
   [86]: https://github.com/sbt/zinc/pull/86
+  
+- sbt.internal.inc.ClassfileManager has been renamed to sbt.internal.inc.ClassFileManager for consistency. [#87][87]
```

Comment:
I don't think internal class name change needs to be included to migration guide.